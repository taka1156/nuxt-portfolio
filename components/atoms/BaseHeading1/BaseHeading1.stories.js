import BaseHeading1 from './BaseHeading1.vue';

export default {
  title: 'atoms/BaseHeading1',
  component: BaseHeading1
};

const Template = () => ({
  components: { BaseHeading1 },
  template: '<base-heading1>ダミーテキスト</base-heading1>'
});

export const Default = Template.bind({});
