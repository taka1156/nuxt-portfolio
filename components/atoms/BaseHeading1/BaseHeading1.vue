<template>
  <div>
    <h1 class="base-heading1--extend">
      <slot />
    </h1>
  </div>
</template>

<script>
export default {
  name: 'BaseHeading1'
};
</script>
