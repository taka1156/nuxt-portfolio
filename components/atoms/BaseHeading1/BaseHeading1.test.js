import { mount } from '@vue/test-utils';
import BaseHeading1 from './BaseHeading1.vue';

describe('BaseImg', () => {
  const baseHeading1 = () =>
    mount(BaseHeading1, {
      slots: {
        default: 'ダミーの見出し'
      }
    });

  it('値がDOMに反映されているか', () => {
    const wrapper = baseHeading1();
    const hTag = wrapper.find('h1');
    expect(hTag.text()).toBe('ダミーの見出し');
    // スナップショット
    expect(wrapper.html()).toMatchSnapshot();
  });
});
