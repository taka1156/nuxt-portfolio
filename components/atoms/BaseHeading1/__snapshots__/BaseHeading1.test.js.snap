// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseImg 値がDOMに反映されているか 1`] = `
"<div>
  <h1 class=\\"base-heading1--extend\\">ダミーの見出し</h1>
</div>"
`;
